Create ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md (#2645)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md

---
date: ""

> title: [UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/y2024] (#7581)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Firmware update process fails if device reports unexpected UPDATE_RESULT
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4050
version: 1018.503.132
---

> [UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/y2024] (#7581)

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, devices reported an unexpected UPDATE_RESULT during the firmware update process, but the update was still considered successful. With this change, the firmware update process now fails if a device reports an unexpected UPDATE_RESULT. This prevents incorrectly reporting a successful firmware update when the device actually encountered an error. Devices which have already reported an unexpected UPDATE_RESULT during a firmware update are not affected, but new firmware updates will now properly fail if this situation occurs.
```

---
date: ""
title: Terminate the firmware update process if an unexpected situation is detected
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4050
version: 1018.503.132
---

> Before the change if a device reported an unexpected status or result during a firmware update,

Reviewer: ```suggestion
Previously, if a device reported an unexpected status or result during a firmware update,
```

> the process would simply wait for a valid state. Now, you can configure LWM2M devices to fail the firmware update

Reviewer: ```suggestion
the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
```
